mysql-test/suite/sys_vars/t/replicate_do_db_basic.test

```diff
@@ -42,4 +58,5 @@ SET @@GLOBAL.replicate_do_db=DEFAULT;
 SELECT @@GLOBAL.replicate_do_db;
 
 --echo # Cleanup.
+RESET REPLICA ALL;
```

Comment:
As a general comment, usually we don't want to reset the slave (nor master) after a test, as it can remove debugging information (e.g. binary logs). Though in this case, I think it makes sense to keep it, as there isn't really any replication being tested, and there shouldn't be any relay logs or anything that would be useful to read through. Just for your future knowledge :smile: